programs/aes/aescrypt2.c

```diff
@@ -447,6 +447,9 @@ int main( int argc, char *argv[] )
     mbedtls_aes_free( &aes_ctx );
     mbedtls_md_free( &sha_ctx );
 
+    if( ret != 0 && ret != 1)
```

Comment:
It probably doesn't matter, but wouldn't be `if( ret != 0 )` sufficient?

Reply:
Yeah, it doesn't matter, and `if( ret != 0)` would be sufficient. It looks a bit funny to have both `!=0` and `!=1`, but it'll work fine. I've no issue to raise here.

However, instead of `1`, we should use `EXIT_FAILURE` in all cases where `1` is destined to be used directly by `exit()`.